test/resources/gatekeeper_config_test/config_no_exempt_ns.yaml

```
@@ -0,0 +1,10 @@
+apiVersion: config.gatekeeper.sh/v1alpha1
+kind: Config
+metadata:
+  name: config
+spec:
+  match:
+  - excludedNamespaces:
+    - nono
+    processes:
+    - audit
```

(Optional) Add trailing whitespace (because for some reason that red circle really irks me 😆 )
```suggestion
    - audit

```